core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -440,6 +444,9 @@ private[spark] class MesosClusterScheduler(
         .mkString(",")
       options ++= Seq("--py-files", formattedFiles)
     }
+    desc.schedulerProperties
+      .filter { case (key, _) => !replicatedOptionsBlacklist.contains(key) }
+      .foreach { case (key, value) => options ++= Seq("--conf", Seq(key, "=\"", value, "\"").mkString("")) }
```

Comment:
Are the quotes inside `--conf key="value"` valid? I know I tried `--conf spark.mesos.executor.cores="/var/spark"` and that failed (the actual value of `spark.mesos.executor.home` contained the quotes.